server/src/main/java/org/apache/cassandra/sidecar/db/SystemViewsDatabaseAccessor.java

```diff
@@ -94,12 +95,24 @@ public Long cdcTotalSpaceBytesSetting() throws SchemaUnavailableException
     public Map<String, String> getSettings(String... names) throws SchemaUnavailableException
     {
         BoundStatement statement = tableSchema.selectSettings().bind(Arrays.asList(names));
+        return querySettings(statement);
+    }
+
+    @NotNull
+    public Map<String, String> getSettings() throws SchemaUnavailableException
```

Comment:
NIT, maybe we can rename it to something more descriptive like `allSettings`
```suggestion
    public Map<String, String> allSettings() throws SchemaUnavailableException
```